src/Symfony/Bundle/FrameworkBundle/Controller/ControllerResolver.php

```diff
@@ -38,15 +38,14 @@ protected function createController($controller)
         if (false === strpos($controller, '::') && 2 === substr_count($controller, ':')) {
             // controller in the a:b:c notation then
             $controller = $this->parser->parse($controller);
-        }
-
-        $resolvedController = parent::createController($controller);
 
-        if (1 === substr_count($controller, ':') && is_array($resolvedController)) {
-            $resolvedController[0] = $this->configureController($resolvedController[0]);
```

Comment:
This did not configure the controller when it's invokable. It's fixed now since it's also going through instantiateController now like all the other cases.